exchanges/kucoin/kucoin_websocket.go

```diff
@@ -94,12 +85,27 @@ var (
 	maxWSOrderbookWorkers = 10
 )
 
+var defaultSubscriptions = subscription.List{
```

Comment:
I'm okay with this one being knocked back, but there is spot candle data support which does work:
```go
	{Enabled: false, Asset: asset.Spot, Channel: subscription.CandlesChannel, Interval: kline.OneMin},
```

```json
 {
      "enabled": true,
      "channel": "candles",
      "asset": "spot",
      "interval": "1m"
     }
```

```
[DEBUG] | WEBSOCKET | 07/08/2024 15:08:55 | Kucoin websocket connection: message received: {"topic":"/market/candles:ATOM-USDT_1min","type":"message","subject":"trade.candles.update","data":{"symbol":"ATOM-USDT","candles":["1723007280","4.9799","4.9803","4.9807","4.9799","37.3","185.77914072"],"time":1723007335395578690}}
```
In GCT we don't do anything with it though, it can be utilised externally, but I'm okay with it not being here

Reply:
I wasn't generally adding available subscriptions to this list currently.
Exception if it's disabled for some sort of documentable reason that might interest people.